### PR TITLE
Changed the ImageBasic atom to always include an optional alt by default

### PR DESCRIPTION
**Pull request by kave**
@jimmynotjim 
@Scotchester 
@kurtw

### CODE REVIEW

**Comment by Scotchester** [comment]
Thanks! While I certainly approve of the goal stated in the title of the PR, I am not qualified to review the code changes.

**Comment by jimmynotjim** [comment]
Ditto. It looks right, but @kurtw or @rosskarchner  should definitely sign off on this one

File: cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html [hunk 1]
```diff
@@ -37,8 +37,8 @@
                 'image': {
                       'upload': {
                                 'url': 'http://placehold.it/300x300',
-                                'alt': ''
-                      }
+                      },
+                      'alt': ''
```
Comment (sebworks) [inline cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html]: Shouldn't this say 'This is a placeholder image' as well?

File: cfgov/v1/models/atoms.py [hunk 2]
```diff
@@ -92,12 +93,4 @@ def clean(self, data):
         if error_dict:
             raise ValidationError("ImageBasicUrlAlt validation errors", params=error_dict)
```
Comment (sebworks) [inline cfgov/v1/models/atoms.py]: why pass in params if you aren't formatting the string?

File: cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html (hunk 1 above)
Comment (kave) [inline cfgov/jinja2/v1/_includes/molecules/image-text-25-75.html]: @sebworks if you want. i think whoever put that there was trying to show case the image as a bg

File: cfgov/v1/models/atoms.py (hunk 2 above)
Comment (kave) [inline cfgov/v1/models/atoms.py]: it carries additional errors resolved by default wagtail error checking in this line.
```
try:
            data = super(ImageBasicUrl, self).clean(data)
        except ValidationError as e:
            error_dict.update(e.params)
```

Comment (sebworks) [inline cfgov/v1/models/atoms.py]: gotcha, thanks man.

**Pull request merged**